src/components/search/Search.js

```diff
@@ -1,20 +1,50 @@
 import "./search.css";
-import x from "../../img/x.svg"
-export function Search(){
+import x from "../../img/x.svg";
+import search from "../../img/search.svg";
+import noresults from "../../img/nores.svg";
+export function Search(open) {
+  function hideSearch() {
+    const searchBackground = document.querySelector(".search-background");
+    searchBackground.classList.add("hide-search");
+  }
 
-return(
-<div class="search-background">
-    <div class="search-content">
-                <div class="close">
-                    <img src={x}></img>
-                </div>
+  return (
+    <div>
+      <div class={`search-background ${open ? "hide-search" : ""}`}>
+        <div class="search-content">
+          <div class="close" onClick={hideSearch}>
+            <img src={x}></img>
```

Comment:
tags like `<img>` that don't have child elements can be added like this: `<img src={x} />`. No need for the closing tab.